docs/v1/api.md

```diff
@@ -1,63 +1,93 @@
+systems are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
 *__Note:__ All names herein are tentative, and should be considered placeholders
-and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized.*
+for purposes of discussion. This note will be removed when names are finalized.*
 
-## `Broker`
+## `Broker` Resource
 
-This resource is created by an administrator to instruct the service catalog's
-controller event loop to do the following:
+A `Broker` represents an entity that provides service classes for use in the
+catalog.
 
-1. Make a request against a given CF service broker's catalog endpoint
+An administrator to creates an instance of the `Broker` resource to indicate
+their intent to consume that broker in the service catalog. 
+
+The catalog controller has a watch on the `Broker` resource.  When the
+controller receives an `ADD` watch event for a new `Broker`, it contacts the
+broker to determine what service classes the broker offers:
+
+1. Makes a request against a given CF service broker's catalog endpoint
    (`GET /v2/catalog`)
-2. Translate the response to a list of `ServiceClass`es
-3. Write each translated `ServiceClass` to stable storage
+2. Translates the response to a list of `ServiceClass`
+3. Creates new `ServiceClass` instances in the API server
 
-*TODO: should we call out what happens when a `Broker` resource is deleted?*
+*TODO: What happens when a `Broker` resource is deleted?*
 
-## `ServiceClass`
+## `ServiceClass` Resource
+
+A `ServiceClass` represents an offering in the service catalog.  A
+`ServiceClass` is not usable directly; an `Instance` of a `ServiceClass` must be
+created to be consumed by an application.
 
 This resource is created by the service catalog's controller event loop after
 it has received a `Broker` resource and successfully called the backing CF
-broker's catalog endpoint. It represents a single (serviceID, planID) pair that
-a CF service broker has made available.
+broker's catalog endpoint.
+
+*TODO: Make explicit the relationship between service classes and plans*
+
+## `Instance` Resource
+
+An `Instance` represents a provisioned instance of a `ServiceClass`, and is the
+entity an application binds to.
+
+A service consumer creates an `Instance` to indicate their intent to provision
+an instance of a service class.  The `Instance` has a reference to the
+`ServiceClass` to provision an instance of.
 
-*TODO: what happens in the below cases?*
+The service catalog controller has a watch on the `Instance` resource and
+receives an `ADD` watch event. The controller then attempts to provision a new
+instance:
 
-1. *The `Broker` that caused the `ServiceClass` was deleted*
-1. *The `ServiceClass` itself was deleted*
+1.  The controller calls the provision endpoint on the broker 
```

Comment:
Just like above you explicitly called the GET /v2/... maybe say here:
PUT /v2/service_instances/:instance_id